src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -65,6 +65,23 @@
   protected String presentLeaderboardSelector = "div[id*='TOP_LEADERBOARD']";
   protected String presentHighImpactSlotSelector = "div[id*='INVISIBLE_HIGH_IMPACT']";
 
+  protected String
+      presentFandomDesktopArticleTopLeaderboardSelector = "div[id*='gpt-top-leaderboard-desktop']";
+  protected String
+      presentFandomDesktopArticleBottomLeaderboardSelector = "div[id*='gpt-bottom-leaderboard-desktop']";
+  protected String
+      presentFandomMobileArticleTopLeaderboardSelector = "div[id*='gpt-top-leaderboard-mobile']";
+  protected String
+      presentFandomMobileArticleBottomLeaderboardSelector = "div[id*='gpt-bottom-leaderboard-mobile']";
+  protected String
+      presentFandomHubTopLeaderboardSelector = "div[id*='gpt-top-leaderboard']";
+  protected String
+      presentFandomHubBottomLeaderboardSelector = "div[id*='gpt-bottom-leaderboard']";
+
+  protected String presentFandomTopBoxadSelector = "div[id*='gpt-top-boxad']";
+  protected String presentFandomBottomBoxadSelector = "div[id*='gpt-bottom-boxad']";
+
+
```

Comment:
What do you think about creating [something similar](https://github.com/Wikia/selenium-tests/blob/master/src/test/java/com/wikia/webdriver/common/contentpatterns/AdsContent.java) for Fandom?